tests/core/full_node/test_block_height_map.py

```diff
@@ -148,8 +149,21 @@ async def test_save_restore(self, tmp_dir, db_version):
                 await conn.execute("DROP TABLE full_blocks")
             await setup_db(db_wrapper)
             await setup_chain(db_wrapper, 10000, ses_every=20, start_height=9970)
+
+            # At this point we should have a proper cache file
+            async with aiofiles.open(tmp_dir / "height-to-hash", "rb") as f:
+                heights = bytearray(await f.read())
+                assert len(heights) == (10000 + 1) * 32
```

Comment:
this also looks like you could just use `Path.getsize()`

Reply:
`object has no attribute 'getsize'`

Reply:
https://docs.python.org/3.11/library/pathlib.html#pathlib.Path.stat?

Reply:
Thanks.